test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,69 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+/**
+ * Unit tests for Utils class
+ * 
+ * @author Vivian
+ */
+public class UtilsTest {
+
+    /**
+     * Test method for {@link seedu.addressbook.common.Utils#isAnyNull(java.lang.Object[])}.
+     */
+    @Test
+    public void testIsAnyNull_noNullArgsReturnFalse() {
+        Object obj1 = new Integer(2);
+        Object obj2 = new String("String object");
+        assertFalse("Should have no null args", Utils.isAnyNull(obj1, obj2));
+    }
+    
+    /**
+     * Test method for {@link seedu.addressbook.common.Utils#isAnyNull(java.lang.Object[])}.
+     */
+    @Test
+    public void testIsAnyNull_hasNullArgsReturnTrue() {
+        Object obj1 = null;
+        Object obj2 = new String("String object");
+        assertTrue("Should have at least 1 null args", Utils.isAnyNull(obj1, obj2));
+    }
+
+    /**
+     * Test method for {@link seedu.addressbook.common.Utils#elementsAreUnique(java.util.Collection)}.
+     */
+    @Test
+    public void testElementsAreUnique_hasNonUniqueElementsReturnFalse() {
+        String string1 = new String("This is string 1");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.